lib/services/local/users_test.go

```diff
@@ -845,3 +848,53 @@ func TestIdentityService_SSODiagnosticInfoCrud(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, &info, infoGet)
 }
+
+func TestIdentityService_UpsertKeyAttestationData(t *testing.T) {
+	identity := newIdentityService(t, clockwork.NewFakeClock())
+	ctx := context.Background()
+
+	for _, tc := range []struct {
+		name             string
+		pubKeyPEM        string
+		expectPubKeyHash string
+	}{
+		{
+			name: "standard public key",
+			pubKeyPEM: `-----BEGIN PUBLIC KEY-----
+MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDCep78YgY5I8RrvhE5zra4k1hx
+JZoZL1NsgqBz/f49OZsck24rcxurnC0lKAJmSGtKZrv54E/XZuPtatUkrXtIFKC6
+shHLLAc/LAVtDX2/E/aLgM0srYtt1/kku9H1C9+Ou7RzOIdblRkNMYcbUOhKBNld
+AnYsqjU9/7IaQSp8DwIDAQAB
+-----END PUBLIC KEY-----`,
+		}, {
+			name: "public key with // in standard hash",
```

Comment:
Should we add a case for a pub key with padding in URLEncoding? Or does this cover that also?

Reply:
Sorry I don't follow, why would there be padding? Do you think we need to `strings.TrimSpace()` the public key DER, or something else?

Reply:
Sorry, I meant padding in the hypothetical b64 URL encoding, which is the `=` problem you had earlier.